packages/shared/src/components/MainLayout.tsx

```diff
@@ -225,7 +225,7 @@ function MainLayoutComponent({
       />
       <main
         className={classNames(
-          'flex flex-row',
+          'flex flex-col tablet:flex-row',
```

Comment:
This could have massive impact on some pages as many use this.
We have to make tripple sure none break, think about devcard, sources, tags, whatever hidden page we have.

Can you just make sure before we merge (and really try to find all pages 🙏 )

Reply:
Yeah, but I realized no pages are using row layout for mobile since it wouldn't fit. With that, definitely going to triple-check all pages.